---
layout: theophylline_style
title: Neurotensin receptor NTS-1 aptamer
date: 2024-05-22 00:00:00
author: Bo Fu, Jiali Wang
categories: Aptamer
tags:
- Category:Mammalian
- GC:58.8
- Length:68
- Named:Neurotensin receptor NTS-1 (NTR)_P19
- Type:Proteins
- Year:2002
type: Proteins
---
<html>

<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="description">Description</p>
<p>Grisshammer, R. et al. reported aptamers with affinity for neurotensin receptor neurotensin receptor NTS-1 in their article published in 2002. The P19 aptamer was named by Grisshammer, R. et al. in the article<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In their work published in 2002, Grisshammer, R. et al. used SELEX to isolate RNA aptamer sequences with affinity for neurotensin receptor NTS-1 from a nucleic acid library containing about 6×10<sup>14</sup> sequences after 7 rounds of selection process. The methodology was based on SELEX procedures described, with the exception that paramagnetic beads and a subtraction component were used<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figure is based on the prediction results of the RNA fold website by online secondary structure prediction tool to draw. The figure shows the secondary structure prediction of the original aptamer sequence. This aptamer contains two stem loops of different lengths. Overall, the predicted secondary structure of this aptamer has quite a number of nucleotides that are not in the classic pairing state<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>5'-AGGACGATGCGGACAGATACGGAACTACAGAGGTCAATTACGGTGGCCACGCCAGACGACTCGCCCGA-3'</p>
<img src="/images/2D/Neurotensin receptor NTS-1_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>



<p class="header_box" id="ligand-recognition">Ligand information</p>

<p class="blowheader_box">SELEX ligand</p>
<p>Neurotensin receptor type 1 is G-protein coupled receptor for the tridecapeptide neurotensin (NTS). Signaling is effected via G proteins that activate a phosphatidylinositol-calcium second messenger system. Signaling leads to the activation of downstream MAP kinases and protects cells against apoptosis (By similarity). NTR undergoes N-glycosylation modification and palmitoylation modification after translation. Among them, palmitoylation modification is required for normal localization at membrane rafts and normal GNA11-mediated activation of down-stream signaling cascades. The palmitoylation level increases in response to neurotensin treatment.-----From <a href="https://www.uniprot.org/uniprotkb/P20789/entry" target="_blank" style="color:#520049; text-decoration: underline;"><b>Uniprot</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">Mass</th>
        <th onclick="sortTable(4)">Protein sequence</th>
        <th onclick="sortTable(5)">PDB ID</th>
        <th onclick="sortTable(6)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">NTR (neurotensin receptor NTS-1)</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P20789/entry" target="_blank" style="color:#520049"><b>P20789</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00001/" target="_blank" style="color:#520049"><b>PF00001</b></a></td>
        <td name="td3">47.055 kDa</td>
        <td name="td4">
        <div class="sequence-container">
          <span class="sequence-text"></span>
          <span class="show-more" onclick="toggleSequence(event)">......</span>
          <span class="full-sequence">MHLNSSVPQGTPGEPDAQPFSGPQSEMEATFLALSLSNGSGNTSESDTAGPNSDLDVNTDIYSKVLVTAIYLALFVVGTVGNSVTAFTLARKKSLQSLQSTVHYHLGSLALSDLLILLLAMPVELYNFIWVHHPWAFGDAGCRGYYFLRDACTYATALNVASLSVERYLAICHPFKAKTLMSRSRTKKFISAIWLASALLAIPMLFTMGLQNRSGDGTHPGGLVCTPIVDTATVKVVIQVNTFMSFLFPMLVISILNTVIANKLTVMVHQAAEQGRVCTVGTHNGLEHSTFNMTIEPGRVQALRHGVLVLRAVVIAFVVCWLPYHVRRLMFCYISDEQWTTFLFDFYHYFYMLTNALFYVSSAINPILYNLVSANFRQVFLSTLACLCPGWRHRRKKRPTFSRKPNSMSSNHAFSTSATRETLY</span>
        </div>
        </td>
        <td name="td5">
        <a href="https://www.rcsb.org/structure/3ZEV" target="_blank" style="color:#520049"><b>3ZEV</b></a>
        </td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/366274" target="_blank" style="color:#520049"><b>366274</b></a></td>
      </tr>
	  </tbody>
  </table>

<p>Some isolated sequences bind to the affinity of the protein<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
      <td name="td0">P19 aptamer</td>
      <td name="td1">5'-AGGACGATGCGGACAGATACGGAACTACAGAGGTCAATTACGGTGGCCACGCCAGACGACTCGCCCGA-3'</td>
      <td name="td2">NTR</td>
      <td name="td3">0.37 nM</td>
    </tr>
     <tr>
      <td name="td0">P108 aptamer</td>
      <td name="td1">5'-AGGACGATGCGGGGAAAAGATGGTATGGGCCCTAGAAAGTCCACCCGGCGCCCAGACGACTCGCCCGA-3'</td>
      <td name="td2">NTR</td>
      <td name="td3">5 nM</td>
    </tr>
     <tr>
      <td name="td0">P101 aptamer</td>
      <td name="td1">5'-AGGACGATGCGGGCGACTCCTGTACTCGAANGACAGACAGTTCCAGCCTCCAGCAGACGACTCGCCCGA-3'</td>
      <td name="td2">NTR</td>
      <td name="td3">19 nM</td>
    </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/P19_aptamer_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>


<p class="blowheader_box">Similar compound(s)</p>                    
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD (Å)</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/3ZEV" target="_blank" style="color:#520049"><b>3ZEV</b></a>-B</td>
      <td name="td1">49.2</td>
      <td name="td2">0</td>
      <td name="td3">Original chain</td>
    </tr>
      <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/4XES" target="_blank" style="color:#520049"><b>4XES</b></a>-A</td>
      <td name="td1">36.6</td>
      <td name="td2">1.7</td>
      <td name="td3">Neurotensin receptor type 1, endolysin chimera</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/6TPN" target="_blank" style="color:#520049"><b>6TPN</b></a>-A</td>
      <td name="td1">32.3</td>
      <td name="td2">2.3</td>
      <td name="td3">Orexin receptor type 2,glga glycogen synthase, Ore</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/7XN9" target="_blank" style="color:#520049"><b>7XN9</b></a>-A</td>
      <td name="td1">30.4</td>
      <td name="td2">2.5</td>
      <td name="td3">Somatostatin receptor type 2,endo-1,4-beta-Xylana</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/8XGR" target="_blank" style="color:#520049"><b>8XGR</b></a>-R</td>
      <td name="td1">28.9</td>
      <td name="td2">2.6</td>
      <td name="td3">Guanine nucleotide-binding protein G(I)/G(S)/G(O)</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/6TPK" target="_blank" style="color:#520049"><b>6TPK</b></a>-A</td>
      <td name="td1">27.5</td>
      <td name="td2">2.2</td>
      <td name="td3">Oxytocin receptor</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/7K15" target="_blank" style="color:#520049"><b>7K15</b></a>-A</td>
      <td name="td1">27.4</td>
      <td name="td2">3.0</td>
      <td name="td3">Leukotriene b4 receptor 1, flavodoxin, leukotriene</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/5K2A" target="_blank" style="color:#520049"><b>5K2A</b></a>-A</td>
      <td name="td1">26.6</td>
      <td name="td2">2.7</td>
      <td name="td3">Adenosine receptor A2A/soluble cytochrome B562 CH</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/7F1T" target="_blank" style="color:#520049"><b>7F1T</b></a>-A</td>
      <td name="td1">25.7</td>
      <td name="td2">3.3</td>
      <td name="td3">C-C motif chemokine 3, C-C chemokine receptor type</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/6LW5" target="_blank" style="color:#520049"><b>6LW5</b></a>-A</td>
      <td name="td1">25.3</td>
      <td name="td2">2.7</td>
      <td name="td3">Soluble cytochrome B562, N-formyl peptide receptor</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/5WB2" target="_blank" style="color:#520049"><b>5WB2</b></a>-A</td>
      <td name="td1">24.3</td>
      <td name="td2">3.2</td>
      <td name="td3">Envelope protein US28, nanobody 7 fusion protein</td>
    </tr>
    </tbody>
  </table>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Generation of RNA aptamers to the G-protein-coupled receptor for neurotensin, NTS-1.</strong></font><br />
Daniels, D. A., Sohal, A. K., Rees, S., & Grisshammer, R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12054450/" target="_blank" style="color:#520049">Analytical biochemistry, 305(2), 214–226. (2002)</a>
<br/>


<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 20);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>