---
layout: theophylline_style
title: SARS-CoV N protein aptamer
date: 2024-05-06 00:00:00
author: Jiali Wang, Bo Fu
categories: Aptamer
tags:
- Category:Viral
- GC:56.00000000000001
- GC:59.040000000000006
- Length:83
- Length:84
- Named:SARS-CoV nucleocapsid (N) protein_aptamer 1
- Named:SARS-CoV nucleocapsid (N) protein_aptamer 2
- Type:Proteins
- Year:2009
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="description">Description</p>
<p>In 2009, Oh, J. W et al. used the SELEX method to select the aptamer with high compatibility for the SARS-CoV nucleocapsid (N) protein. The aptamer antibody mixed immunoassay method they studied may be used for rapid and sensitive detection of SARS-CoV N protein, which has potential application value for improving the treatment and prognosis of SARS<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2009, Oh, J. W et al. and colleagues carried out SELEX screening of an RNA aptamer pool containing approximately 10 <sub>14</sub> random, 40-nt long aptamers to isolate those binding to the SARS-CoV N protein. The process involved nine rounds of SELEX: five preliminary rounds and four further rounds under more stringent conditions (with a fivefold reduction in the amount of N protein). After nine rounds of screening, the cDNA of the selected RNA population was cloned, and 17 independent clones were randomly selected for sequencing. The team further verified the binding specificity and affinity of the selected aptamers to the N protein using electrophoretic mobility shift assays (EMSAs) and surface plasmon resonance (SPR) analysis<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figure is based on the article by ribodraw tool to draw. Aptamer 1 binds to SARS-CoV nucleocapsid (N) protein. Aptamer 1 was named by Oh, J. W et al<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>5'-GGGAGAGCGGAAGCGUGCUGGGCCUGUCGUUCGCUGUCUUGCUACGUUACGUUACACGGUUGGCAUAACCCAGAGGUCGAUGG-3'</p>
<img src="/images/2D/aptamer_1_2D1.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>





<p class="header_box" id="ligand-recognition">Ligand information</p> 

<p class="blowheader_box">SELEX ligand</p>
<p>Some isolated sequences bind to the affinity of the protein<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">apatamer 1</td>
        <td name="td1">5'-GGGAGAGCGGAAGCGUGCUGGGCCUGUCGUUCGCUGUCUUGCUACGUUACGUUACACGGUUGGCAUAACCCAGAGGUCGAUGG-3'</td>
        <td name="td2">SARS-CoV N protein</td>
        <td name="td3">1.65 Â± 0.41 nM (equilibrium dissociation constant)<br>0.81 nM (apparent dissociation constant)</td>
      </tr>
	  </tbody>
     <tbody>
      <tr>
        <td name="td0">apatamer 2</td>
        <td name="td1">5'-GGGAGAGCGGAAGCGUGCUGGGCCUCAUUACACACAUCUCACGGGAGACAUAGCUGACGAUAUCCAUAACCCAGAGGUCGAUGG-3'</td>
        <td name="td2">SARS-CoV N protein</td>
        <td name="td3">3.35 nM (apparent dissociation constant)</td>
      </tr>
	  </tbody>
    </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/SARS_CoV_N_protein_SELEX_ligand.svg" alt="drawing" style="width:1000px;height:350px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>



<p class="blowheader_box">Structure ligand</p>
<p>Coronavirus (CoV) nucleocapsid (N) proteins have 3 highly conserved domains. The N-terminal domain (NTD) (N1b), the C-terminal domain (CTD)(N2b) and the N3 region. The N1b and N2b domains from SARS CoV, infectious bronchitis virus (IBV), human CoV 229E and mouse hepatic virus (MHV) display similar topological organisations. N proteins form dimers, which are asymmetrically arranged into octamers via their N2b domains. Domains N1b and N2b are linked by another domain N2a that contains an SR-rich region in which phosphorylation of specific serine residues allows the N protein to associate with the RNA helicase DDX1 permitting template read-through, and enabling the transition from discontinuous transcription of subgenomic mRNAs (sgmRNAs) to continuous synthesis of longer sgmRNAs and genomic RNA (gRNA). It has been shown that N proteins interact with nonstructural protein 3 (NSP3) and thus are recruited to the replication-transcription complexes (RTCs).-----From <a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00937/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">Mass</th>
        <th onclick="sortTable(4)">Protein sequence</th>
        <th onclick="sortTable(5)">PDB ID</th>
        <th onclick="sortTable(6)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">SARS-CoV nucleocapsid (N) protein</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P59595/entry" target="_blank" style="color:#520049"><b>P59595</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00937/" target="_blank" style="color:#520049"><b>PF00937</b></a></td>
        <td name="td3">13.913 kDa</td>
        <td name="td4">
        <div class="sequence-container">
          <span class="sequence-text"></span>
          <span class="show-more" onclick="toggleSequence(event)">......</span>
          <span class="full-sequence">NTASWFTALTQHGKEELRFPRGQGVPINTNSGPDDQIGYYRRATRRVRGGDGKMKELSPRWYFYYLGTGPEASLPYGANKEGIVWVATEGALNTPKDHIGTRNPNNNAATVLQLPQGTTLPKGFYA</span>
        </div>
        </td>
        <td name="td5"><a href="https://www.rcsb.org/structure/2OFZ" target="_blank" style="color:#520049"><b>2OFZ</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/nuccore/AY278741" target="_blank" style="color:#520049"><b>AY278741</b></a></td>
      </tr>
	  </tbody>
  </table>

<p class="blowheader_box">Similar compound(s)</p>                    
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD (Ã)</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/2OFZ" target="_blank" style="color:#520049"><b>2OFZ</b></a>-A</td>
        <td name="td2">26.0</td>
        <td name="td3">0</td>
        <td name="td4">Nucleocapsid protein</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/7N0R" target="_blank" style="color:#520049"><b>7N0R</b></a>-A</td>
        <td name="td2">24.9</td>
        <td name="td3">0.7</td>
        <td name="td4">Nucleocapsid protein</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/7N0R" target="_blank" style="color:#520049"><b>7N0R</b></a>-B</td>
        <td name="td2">24.8</td>
        <td name="td3">0.8</td>
        <td name="td4">Nucleocapsid protein</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/7XX1" target="_blank" style="color:#520049"><b>7XX1</b></a>-B</td>
        <td name="td2">24.5</td>
        <td name="td3">0.8</td>
        <td name="td4">Nucleocapsid protein</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/6VYO" target="_blank" style="color:#520049"><b>6VYO</b></a>-C</td>
        <td name="td2">24.5</td>
        <td name="td3">0.6</td>
        <td name="td4">Nucleocapsid protein</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/2OG3" target="_blank" style="color:#520049"><b>2OG3</b></a>-A</td>
        <td name="td2">24.3</td>
        <td name="td3">0.4</td>
        <td name="td4">Nucleocapsid protein</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/9EZB" target="_blank" style="color:#520049"><b>9EZB</b></a>-A</td>
        <td name="td2">24.3</td>
        <td name="td3">0.9</td>
        <td name="td4">Nucleocapsid protein</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/7N3C" target="_blank" style="color:#520049"><b>7N3C</b></a>-C</td>
        <td name="td2">24.3</td>
        <td name="td3">0.9</td>
        <td name="td4">S24-202 FAB heavy chain</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/9EVY" target="_blank" style="color:#520049"><b>9EVY</b></a>-D</td>
        <td name="td2">24.3</td>
        <td name="td3">0.9</td>
        <td name="td4">Nucleocapsid protein</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/6VYO" target="_blank" style="color:#520049"><b>6VYO</b></a>-A</td>
        <td name="td2">24.3</td>
        <td name="td3">1.2</td>
        <td name="td4">Nucleocapsid protein</td>
      </tr>
    </tbody>
  </table>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] RNA aptamer-based sensitive detection of SARS coronavirus nucleocapsid protein.</strong></font><br />
Ahn, D. G., Jeon, I. J., Kim, J. D., Song, M. S., Han, S. R., Lee, S. W., Jung, H., & Oh, J. W.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/19684916/" target="_blank" style="color:#520049">The Analyst, 134(9), 1896â1901. (2009)</a>
<br/>


<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // å±å¼åæé®ææ¬åå
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // å±å¼åæ¾ç¤º "..."
      } else {
        showMoreText.textContent = '......';  // æ¶èµ·åæ¾ç¤º "......"
      }
    }

    // é¡µé¢å è½½æ¶ï¼éå¶åºåææ¬ä¸º50ä¸ªå­ç¬¦
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 20);  // åªæ¾ç¤ºå50ä¸ªå­ç¬¦
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>